lib/index.js

```diff
@@ -86,27 +87,38 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
+  var opts = this.options;
+  var eventProperties = track.properties();
+
+  // Optimizely expects revenue only passed through Order Completed events
```

Comment:
@hankim813  Refactored and left comments, but let me know if you were thinking of refactoring this differently.